signin/orgreg.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet1.css">
+    <title>OrganisationRegister</title>
+</head>
+
+<body>
+    <div class="class1">
+        <div class="start1">
+    <div class="start" id="start">
+    <img src="logo/logo.png" alt="Image" class="img1">
```

Comment:
alt attributes should be more descriptive, the img tag already identifies the image so the alt tag is to describe what's featured in the image so screen readers, web crawlers and search engines can index it